schema.graphql

```diff
@@ -0,0 +1,15 @@
+# A schema is a collection of type definitions (hence "typeDefs")
+# that together define the "shape" of queries that are executed against
+# your data.
+# A GraphQL schema defines what kind of object can be fetched from a service, and what fields it has. 
+# The schema can be defined using GraphQL Schema Definition Language. 
+
+# Here, the query contains a longitude and latitude attributes that returns a string value.
+
+type Query {
+  address(id: ID!): Address
```

Comment:
I would like to hear your thoughts on the decision to use an ID for an arbitrary address name

Reply:
Oh there is no specific reason, just couldn't come up with better property name, i can use `name` instead.

Reply:
<img width="880" alt="Screen Shot 2022-08-16 at 2 55 15 AM" src="https://user-images.githubusercontent.com/22843777/184816418-4f69881d-9ff2-4aaf-8a87-f1a58b335fe6.png">
Made this change, can push tomorrow